.github/workflows/build-macos.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v6
         with:
-          go-version: '1.25.1'
+          go-version: '1.25.3'
```

Comment:
I think this will always use the latest stable release (1.25.x), which I think is a better idea than picking one specific version:

```suggestion
          go-version: '1.25'
```

Reply:
Are you sure about this? We'd get auto switched to every next major Go version when it's released and hits `stable` :thinking:

Reply:
Oh no please don't use `stable`. It will just suddenly break CI. Yes we want to update TinyGo whether a new stable Go release happens, but sometimes things get delayed and I would really want to have usable CI during that time.

Reply:
Looked again, and behold:

```
Found in cache @ /Users/runner/hostedtoolcache/go/1.25.1/arm64
```

So `1.25` just uses whichever is cached. I will have to go back to `1.25.3`